src/Compilers/Core/Portable/Operations/OperationCloner.cs

```diff
@@ -662,5 +662,7 @@ public override IOperation VisitUsingDeclaration(IUsingDeclarationOperation oper
         {
             return new UsingDeclarationOperation(Visit(operation.DeclarationGroup), operation.IsAsynchronous, ((Operation)operation).OwningSemanticModel, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
         }
+
+        // TODO2
```

Comment:
Could you be more specific and create an issue if more work needs to be done at a later point.